src/main/java/ch/njol/skript/Skript.java

```diff
@@ -584,7 +590,7 @@ public void run() {
 					info("Preparing Skript for testing...");
 					tainted = true;
 					try {
-						getAddonInstance().loadClasses("ch.njol.skript", "tests");
+						getAddonInstance().loadClasses("ch.njol.skript.test", "runner");
```

Comment:
```suggestion
						getAddonInstance().loadClasses("ch.njol.skript.test.runner");
```

Reply:
The sub packages VarArgs cannot be null. This will throw an assertion error.

Reply:
I don't think it would be null, just empty. The javadoc for the method  says that it being empty means all subpackages will be loaded. Since there is just one subpackage in this case we can move it to the base package parameter